var/spack/repos/builtin/packages/cntk/package.py

```diff
@@ -34,7 +34,9 @@ class Cntk(Package):
     url      = "https://github.com/Microsoft/CNTK/archive/v2.0.beta15.0.tar.gz"
 
     version('master', git='https://github.com/Microsoft/CNTK.git', branch='master')
-    version('2.0.rc1', url='https://github.com/Microsoft/CNTK/archive/v2.0.rc1.tar.gz')
+    version('2.0', '8038780f1169ceea578e5ef4d69e4c6f', url='https://github.com/Microsoft/CNTK/archive/v2.0.tar.gz')
+    version('2.0.rc3', 'ccd583d4095e02d8ba4c77043e9859d5', url='https://github.com/Microsoft/CNTK/archive/v2.0.rc3.tar.gz')
+    version('2.0.rc1', 'cdc02a1754cb80999bb7edec3c3ad164', url='https://github.com/Microsoft/CNTK/archive/v2.0.rc1.tar.gz')
```

Comment:
We don't normally add release candidates of a package to Spack. Do you really need these? Also, you don't need to specify the URL for each version, Spack can guess them correctly. Just update `url` above to one of these URLs and you should be good to go. You can run `spack info cntk` to check.